qml/LogsPane.qml

```diff
@@ -298,6 +303,14 @@ Rectangle
 			spacing: logStyle.generic.layout.headerButtonSpacing
 			height: parent.height
 
+			DefaultLabel
+			{
+				id: labelFilter
+				text: qsTr("Filter:")
+				verticalAlignment: Text.AlignVCenter
+				width: 50
+			}
```

Comment:
"Filter" label looks like another button the user can click. We could add more info to tooltip. and I think we should make the background color a little bit more darker for toggled state